.changeset/tall-carrots-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
```

Comment:
Let's make this a patch (this was a bug). If we don't consider it a bugfix then it's technically a breaking change

Reply:
Done in cbd60b7